src/buffers.jl

```diff
@@ -2,9 +2,6 @@ const MPIInteger = Union{Int8, UInt8, Int16, UInt16, Int32, UInt32, Int64, UInt6
 const MPIFloatingPoint = Union{Float32, Float64}
 const MPIComplex = Union{ComplexF32, ComplexF64}
 
-const MPIDatatype = Union{Int8, UInt8, Int16, UInt16,
-                          Int32, UInt32, Int64, UInt64,
-                            Float32, Float64, ComplexF32, ComplexF64}
```

Comment:
Can we keep this for now? These are the "native" MPI types (i.e. the ones that exist out of the box)

Reply:
Sure.